### ISSUE TITLE
Reply sometimes uses e-mail address of first mail in a conversation

### ISSUE THREAD

**Issue by mrusme** [CTemplar/webclient#1197]
### Your environment

Latest Electron app as well as web UI in Firefox (latest)

### Steps to reproduce

- Have someone send you an e-mail, e.g. from seymour@butz.com
- Reply to that email but change the To: address to someone else, e.g. hugh@jass.com
- Have hugh@jass.com reply to your mail
- Click reply on hugh@jass.com's mail

### Expected behavior

The new mail reply should show hugh@jass.com in the "To:" field

### Actual behavior

*Sometimes* the new mail reply has seymour@butz.com in the "To:" field, even though the last mail in that conversation is from hugh@jass.com

**Comment by The-Hidden-Hand** [comment]
Dear @mrusme,

We have been following the steps and couldn't reproduce it, even once.

How often is this happening to you? Did you find a way to reproduce it always?

**Comment by mrusme** [comment]
@The-Hidden-Hand thanks for getting back! I had this multiple times in a mail conversation, where the way I described was exactly how the communication was initiated. Could there be any mail header involved apart from "Reply-To" (which I checked, wasn't there) that might lead to this behavior?

While I cannot send you the whole set of mails I can offer to send some meta-info/headers that might help find out was happening there. Alternatively feel free to close this ticket for now and as soon as I see this re-occur in a new (active) conversation I can get back here and provide further info with the possibility to test things.

Thank you!
